embulk-jruby-extension/src/main/java/org/jruby/util/Temporal.java

```diff
@@ -0,0 +1,90 @@
+package org.jruby.util;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+public class Temporal // TODO better naming
+{
+    // @see https://github.com/jruby/jruby/blob/master/core/src/main/java/org/jruby/RubyTime.java#L1366
+    public static Temporal newTemporal(ParsedValues values)
+    {
+        long sec;
+        long nsec = 0;
+
+        if (values.hasSeconds()) {
+            if (values.has(values.sec_fraction)) {
+                nsec = values.sec_fraction * (int)Math.pow(10, 9 - values.sec_fraction_size);
+            }
+
+            if (values.has(values.seconds_size)) { // Rational
+                sec = values.seconds / (int)Math.pow(10, values.seconds_size);
+            } else { // int
+                sec = values.seconds;
+            }
+
+        } else {
+            DateTimeZone dtz = DateTimeZone.UTC;
+
+            int year;
+            if (values.has(values.year)) {
+                year = values.year;
+            } else {
+                year = 1970;
+            }
+
+            // set up with min values and then add to allow rolling over
+            DateTime dt = new DateTime(year, 1, 1, 0, 0, 0, 0, DateTimeZone.UTC);
```

Comment:
This setting `1970-01-01 00:00:00` is correct behavior? Is this same behavior with CRuby?